docs/guide/basic-usage.md

```diff
@@ -30,7 +30,7 @@ Testing
 -----------
 
 Define a `tests` alias in your console config. For example, for the `basic` project template, this should be added
-to the `console.php` configuration: `Yii::setAlias('tests', __DIR__ . '/../tests');`
+to the `console.php` configuration: `Yii::setAlias('tests', __DIR__ . '/tests/codeception');`
```

Comment:
note that there is `dirname(__DIR__)` which is basically `__DIR__ . '/../'`. so adding dirname here would be fine too.